compiler/vconone/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if (NOT VCONONE_VERSION)
-  set(VCONONE_VERSION 0x00000000000c0001)
+  set(VCONONE_VERSION 0x00000000000d0001)
```

Comment:
@seanshpark Maybe `0x00000000000d0000`?

Reply:
LSB `0001` is major,  `000d` is 13. so current is 1.13.0